misc/CFGs/CFG.txt

```diff
@@ -0,0 +1,89 @@
+**************************************************************
+CFG
+**************************************************************
+
+Program --> Start $
+
+Start --> FunctionDcl Start
+        | VarDcl Start
+        | EPSILON
+
+Operator --> BracketL
+           | BracketR
+           | ParenL
+           | ParenR
+           | CurlyL
+           | CurlyR
+           | =
+           | ::
+           | <
+           | >
+
+CurlyL   --> {
+CurlyR   --> }
+ParenL   --> (
+ParenR   --> )
+BracketL --> [
+BracketR --> ]
+
+Type --> number
+       | text
+       | tuple
+       | list
+       | dict
+       | char
```

Comment:
skal vi have char? jeg tænkte at en char bare er en streng på 1 "character" uden at char faktisk er accessable fra en bruger synsvinkel. For simplicitet :P